app/components/support/school_details_summary_list_component.rb

```diff
@@ -1,11 +1,32 @@
 class Support::SchoolDetailsSummaryListComponent < ResponsibleBody::SchoolDetailsSummaryListComponent
   def rows
-    super +
-      (headteacher.present? ? [headteacher_row] : [])
+    array = super
+    array << headteacher_row if headteacher.present?
+    array.insert(array.find_index { |row| row[:key] == 'Can place orders?' }, mno_row)
```

Comment:
can we only show this row if the `mno_feature_flag` is enabled?

Reply:
done